lib/class/api.class.php

```diff
@@ -118,6 +118,47 @@ public static function set_filter($filter,$value)
         return true;
     } // set_filter
 
+
+    /**
+     * logout
+     *
+     * This function allows the user to destroy the auth key
+     * Takes auth key
+     * @param array
+     * @return boolean
+     */
+    public static function logout($input)
+    {
+        $authKey = $input['auth'];
+
+        //Whatever format the user wants
+        $outputFormat = $input['format'];
+
+        // Log the attempt
+        debug_event('API', "Destroy Auth Key Attempt, IP:$ip Key:$authKey", 5);
+
+        if(isset($authKey)){
+            if(Session::destroy($authKey)) {
+                if ($outputFormat == 'json') {
+                    echo JSON_Data::single_string('success');
+                }
+                else {  // Defaults to XML
+                    echo XML_Data::single_string('success');
```

Comment:
Why not returning the same kind of structure as in `handshake` method, with `NULL` `session_expires` and this kind of thing?